docs/guides/integration-prebid.md

```diff
@@ -21,33 +21,67 @@ sidebar_position: 04
+
+| Step | Action | Link to Instructions |
+| --- | --- | --- |
+| 1 | Send a server-side API call to generate or refresh the UID2 token.  | [UID2 User ID Submodule](#uid2-user-id-submodule) |
+| 2 | Store the response value, so that the Prebid module can manage token refresh as well as opt-out if needed. | [Client Refresh Mode](#client-refresh-mode) |
 
 ## UID2 User ID Submodule
 
-UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module operates in Client Refresh mode.
+UID2 requires initial tokens to be generated server-side. You can do this by calling one of the following endpoints:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) to generate a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) to refresh an existing token.
+
+The UID2 module handles storing, providing, and optionally refreshing UID2 tokens. The module operates in Client Refresh mode.
 
 >**Important:** UID2 is not designed to be used where GDPR applies. The module checks the passed-in consent data, and does not operate if the `gdprApplies` flag is set to `true`.
 
 ## Client Refresh Mode
 
-In Client Refresh mode, the full response body from the UID2 [POST /token/generate](../endpoints/post-token-generate.md) or [POST /token/refresh](../endpoints/post-token-refresh.md) endpoint must be provided to the module. As long as the refresh token remains valid, the module refreshes the UID2 token (advertising token) as needed.
+You must provide the Prebid module with the full response body from the applicable endpoint:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+
+As long as the refresh token remains valid, the module refreshes the UID2 token as needed.
+
+### Response Storage Options
+
+When you configure the module to use Client Refresh mode, you must choose **either** of the following options for storing the API response information.
+
+| Option | Details | Use Case | 
+| --- | --- | --- |
+| Set `params.uid2Cookie` to the name of the cookie that contains the response body as a JSON string. | See [Client Refresh Cookie Example](#client-refresh-cookie-example). | {**GWH_SW data to come**) |
+| Set `params.uid2Token` to the response body as a JavaScript object. | See [Client Refresh uid2Token Example](#client-refresh-uid2token-example). | {**GWH_SW data to come**) |
+
+(**GWH_SW query_01: I've been asked to provide information about why the publisher might choose one or the other. For example, because if the cookie size is too big it's a problem and so we offer a local storage option? She said there was feedback on the cookie size, that it was too large, so as a solution it could be set in local storage. Is this correct? Or are there other reasons they might choose one or the other of these two options?**)
```

Comment:
That's right. For uid2cookie, the use case would be:

_If your cookie has space left to store the response body and requires minimum changes to complete the integration._ 

For UID2 Token it would be:

_If you do not have space left for cookie and prefer the flexibility to manage the storage of the response body yourself, provide the response body via `params.uid2Token`_

Reply:
Done. see update. Thx!